x-pack/plugins/ingest/index.ts

```diff
@@ -20,26 +20,29 @@ export function infra(kibana: any) {
     uiExports: {
       app: {
         description: 'Explore your infrastructure',
-        icon: 'plugins/infra/images/infra_mono_white.svg',
-        main: 'plugins/infra/app',
+        icon: 'plugins/ingest/public/images/infra_mono_white.svg',
```

Comment:
```suggestion
        icon: plugins/ingest/images/infra_mono_white.svg',
```

Seeing broken links on the icons when running. I think the `public` is not needed. Seemed to fix it on my end.